packages/core/upload/admin/src/utils/updateUrlForCaching.js

```diff
@@ -0,0 +1,15 @@
+import { URL } from 'url';
```

Comment:
`URL` is available in the browser, i don't think we need to import it :)

Reply:
Right, I forgot where this was running. This was unnecessary and has been removed.